src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding: 10px;
+
+  @for $month-length from 28 through 31 {
+    & :nth-child(n + 1 + #{$current-length}) {
+      display: none;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      content: "#{$index}";
+      display: block;
+      line-height: 100px;
+      text-align: center;
+    }
+  }
+
+  &__day {
+    @include squre($cell-size);
+    margin: 0 $gap-between-size $gap-between-size 0;
+    box-sizing: border-box;
```

Comment:
move it to the * selector)

Reply:
I have already styled the element .calendar with styles "box-sizing: content-box", so styling "box-sizing: border-box" only applies to calendar__day